pkg/proxy/userspace/roundrobin.go

```diff
@@ -34,8 +34,10 @@ import (
 )
 
 var (
+	// ErrMissingServiceEntry holds the error message for when the service entry is missing
 	ErrMissingServiceEntry = errors.New("missing service entry")
-	ErrMissingEndpoints    = errors.New("missing endpoints")
+	// ErrMissingEndpoints holds the error message for when the endpoints are missing
+	ErrMissingEndpoints = errors.New("missing endpoints")
```

Comment:
should go.fmt warning about this? it seems it has not the same indentation

Reply:
The comment makes it not have to